app/controllers/assignments_controller.rb

```diff
@@ -55,7 +55,7 @@ def show
 
   # rubocop:disable MethodLength
   def filter_repos
-    return unless @organization.roster
+    return unless @organization.roster || search_assignments_enabled?
```

Comment:
Do you think this should be `@organization.roster && search_assignments_enabled?` instead? Otherwise this feature will technically be available for any assignment with an organization

Reply:
Good point 👍